src/SALib/sample/sobol.py

```diff
@@ -0,0 +1,234 @@
+    rows.
+    These model inputs are intended to be used with
+    :func:`SALib.analyze.sobol.analyze`.
+
+    Notes
+    -----
+    The initial points of the Sobol' sequence has some repetition (see Table 2
+    in Campolongo [1]_), which can be avoided by scrambling the sequence.
+
+    Another option, not recommended and available for educational purposes,
+    is to use the `skip_values` parameter.
+    Skipping values reportedly improves the uniformity of samples.
+    But, it has been shown that naively skipping values may reduce accuracy,
+    increasing the number of samples needed to achieve convergence
+    (see Owen [2]_).
+
+    Parameters
+    ----------
+    problem : dict
+        The problem definition
+    N : int
+        The number of samples to generate.
+        Ideally a power of 2 and <= `skip_values`.
+    calc_second_order : bool, optional
+        Calculate second-order sensitivities. Default is True.
+    scramble : bool, optional
+        If True, use LMS+shift scrambling. Otherwise, no scrambling is done.
+        Default is True.
+    skip_values : int, optional
+        Number of points in Sobol' sequence to skip, ideally a value of base 2.
+        It's recommended not to change this value and use `scramble` instead.
+        Default is 0.
+    seed : {None, int, `numpy.random.Generator`}, optional
+        If `seed` is None the `numpy.random.Generator` generator is used.
+        If `seed` is an int, a new ``Generator`` instance is used,
+        seeded with `seed`.
+        If `seed` is already a ``Generator`` instance then that instance is
+        used. Default is None.
+
+    References
+    ----------
+    .. [1] Campolongo, F., Saltelli, A., Cariboni, J., 2011.
+           From screening to quantitative sensitivity analysis.
+           A unified approach.
+           Computer Physics Communications 182, 978–988.
+           https://doi.org/10.1016/j.cpc.2010.12.039
+
+    .. [2] Owen, A. B., 2020.
+           On dropping the first Sobol' point.
+           arXiv:2008.08051 [cs, math, stat].
+           Available at: http://arxiv.org/abs/2008.08051
+           (Accessed: 20 April 2021).
+
+    .. [3] Saltelli, A., 2002.
+           Making best use of model evaluations to compute sensitivity indices.
+           Computer Physics Communications 145, 280–297.
+           https://doi.org/10.1016/S0010-4655(02)00280-1
+
+    .. [4] Sobol', I.M., 2001.
+           Global sensitivity indices for nonlinear mathematical models and
+           their Monte Carlo estimates.
+           Mathematics and Computers in Simulation,
+           The Second IMACS Seminar on Monte Carlo Methods 55, 271–280.
+           https://doi.org/10.1016/S0378-4754(00)00270-6
+    """
+    D = problem['num_vars']
+    groups = _check_groups(problem)
+
+    # Create base sequence - could be any type of sampling
+    qrng = qmc.Sobol(d=2*D, scramble=scramble, seed=seed)
+
+    # fast-forward logic
+    if skip_values > 0:
```

Comment:
Is it acceptable if `skip_values > 0` and `scramble == True`?

Reply:
Whatever the answer, it would be worthwhile to add it to the docstring I think

Reply:
Yes we can combine both. I added a note in the docstrings as suggested.